uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/option-group-mock-data.js

```diff
@@ -0,0 +1,328 @@
+define([
```

Comment:
that will unnecessary bloat number of files and managing their inclusion instead of having a single file and obtain relevant data from them